sunxspex/thermal_spectrum.py

```diff
@@ -10,18 +10,14 @@
 from sunxspex.io import chianti_kev_line_common_load_light, load_xray_abundances
 from sunxspex.utils import get_reverse_indices
 
-SSWDB_XRAY_CHIANTI = os.path.expanduser(os.path.join("~", "ssw", "packages",
-                                                     "xray", "dbase", "chianti"))
-FILE_IN = os.path.join(SSWDB_XRAY_CHIANTI, "chianti_lines_1_10_v71.sav")
 
-
-class ChiantiThermalSpectrum():
+class ChiantiThermalSpectrum:
     """
     Class for evaluating solar X-ray thermal spectrum using CHIANTI data.
     """
     @u.quantity_input(energy_edges=u.keV)
-    def __init__(self, energy_edges, linefile=FILE_IN,
-                 xray_abundance_file=None, abundance_type=None,
+    def __init__(self, energy_edges,
```

Comment:
There might be cases where we want to use non-default files @DanRyanIrish @hayesla ?

Reply:
You can override files using the [`manager.override_file`](https://docs.sunpy.org/en/latest/api/sunpy.data.data_manager.DataManager.html#sunpy.data.data_manager.DataManager.override_file) context manager, so it would be something like:

```python
from sunpy.data import manager

with manager.override_file("xray_abundances", url=...):
    sunxspex.whatever()
```

Reply:
I like this!

Reply:
Ah, I see this is the same comment I made above.  I agree with @samaloney that it's very possible users will want to use their own local versions of this file.  The solution @Cadair suggests looks elegant.  But I don't think it's intuitive since most of the X-ray community is not very familiar with Python.  So we need to document how to do this somewhere.  At least in a docstring here.  Probably also in a guide/tutorial at a later date.

Reply:
for sure - I think an example of this would be good. I guess most users wouldn't have known you could do this in the IDL version anyway :P 

But yes examples all the way 💯